compiler/parser/src/string_parser.rs

```diff
@@ -358,11 +358,7 @@ impl<'a> StringParser<'a> {
                 }
             }
         }
-        Err(if expression.trim().is_empty() {
-            FStringError::new(EmptyExpression, self.get_pos()).into()
-        } else {
-            FStringError::new(UnclosedLbrace, self.get_pos()).into()
-        })
+        Err(FStringError::new(UnclosedLbrace, self.get_pos()).into())
```

Comment:
If we reach here, that means we didn't encounter `}`. That means `{` is not closed.

Reply:
In cpython:

```
>>> f"{   "
  File "<stdin>", line 1
SyntaxError: f-string: expecting '}'
```